client/web/package.json

```diff
@@ -4,6 +4,10 @@
   "version": "1.10.1",
   "description": "The Sourcegraph web app",
   "license": "Apache-2.0",
+  "dependencies": {
+    "@sourcegraph/build-config": "0.0.1",
+    "@sourcegraph/shared": "1.0.0"
+  },
```

Comment:
I had to declare dependencies on local packages that `client/web` uses in order for them to be linked under node_modules. There might be a way to avoid having to declare the version here.